lib/mongo/crypt/binary.rb

```diff
@@ -142,7 +142,7 @@ def ref
       def self.wrap_string(str)
         binary_p = Binding.mongocrypt_binary_new_from_data(
           FFI::MemoryPointer.from_string(str),
-          str.length,
+          str.bytesize,
```

Comment:
Great catch.

Reply:
Could you also check the remaining code under crypt/ to see if this change needs to be made elsewhere?